src/color-slider/color-slider.js

```diff
@@ -30,15 +32,17 @@ export default class ColorSlider extends HTMLElement {
 		this._el.slider.addEventListener("input", this);
 		this._el.spinner.addEventListener("input", this);
```

Comment:
I wonder if we should move these lines to `connectedCallback()` since we remove the corresponding listeners inside `disconnectedCallback()`? 🤔

Reply:
Sounds good

Reply:
If you like, I can commit and push this and other changes we agree on.

Reply:
Yes please!